File: app/src/foss/kotlin/com/microsoft/appcenter/analytics/Analytics.java
package com.microsoft.appcenter.analytics;

import com.microsoft.appcenter.AppCenterService;
import java.util.Map;

public class Analytics implements AppCenterService {
  public static void trackEvent(String name, Map<String, String> properties) {
  }

  public static void trackEvent(String name, Map<String, String> properties, int flags) {
  }

  public static void trackEvent(String name, EventProperties properties) {
  }

  public static void trackEvent(String name, EventProperties properties, int flags) {
  }
}

File: app/src/foss/kotlin/com/microsoft/appcenter/analytics/Analytics.kt
package com.microsoft.appcenter.analytics

import com.microsoft.appcenter.AppCenterService

object Analytics : AppCenterService {

    @JvmStatic
    fun trackEvent(name: String, properties: Map<String, String>) {

    }

    @JvmStatic
    fun trackEvent(name: String, properties: Map<String, String>, flags: Int) {

    }

    @JvmStatic
    fun trackEvent(name: String, properties: EventProperties) {

    }

    @JvmStatic
    fun trackEvent(name: String, properties: EventProperties, flags: Int) {

    }

}
